src/cpu/valuepred/ValuePredictor.py

```
@@ -0,0 +1,46 @@
+    tagWidth = Param.Int(16, "tag-width")
+    logESTBEntrys = Param.Int(7, "log 2 of ES table entry counts")
+    logMaxConfidence = Param.Int(5, "log 2 of max confidence number")
+    thresholdPercent = Param.Float(0.25, "threshold percent of confidence")
+
+    # inflight window configuration
+    idealWindow = Param.Bool(True, "The key in the ideal window is a 64-bit pc, not hashed")
+    inflightWindowTagLength = Param.Int(64, "inflight window tag length")
+
+    # about update strategy
+    # still not use
+    enableTimeMsgInUpdate = Param.Bool(True, "enable use instruction"
```

Ruff (0.15.2)

[error] `Param` may be undefined, or defined from star imports

(F405)